sandbox/sandbox.js

```diff
@@ -76,26 +83,27 @@ Sandbox = {
     },
 
     showResultCode: function () {
-
+        delete Sandbox.formOptions.deliveryOptions;
         code = '<script>' +
             '\n' +
             'var data = ' +
-            JSON.stringify(Sandbox.formOptions, null, '    ') +
+            JSON.stringify(Sandbox.formOptions, null, '  ') +
             ';\n' +
             'MyParcel.init(data);' +
             '\n' +
-            '<script>';
+            '</script>';
 
         $('#result_code').html(Sandbox.htmlEncode(code));
     },
 
     showResultCheckout: function () {
         MyParcel.init(Sandbox.formOptions);
+        Prism.highlightAll();
```

Comment:
Dit komt van wc, toch?

Reply:
Heb ik zelf toegevoegd en is alleen in sandbox.js. De syntax highlighting (Prism.js) in de code preview werd niet meer toegepast na iets te veranderen en nu wel 👍